tests/zenko_e2e/backbeat/tests/api/metrics.js

```diff
@@ -0,0 +1,200 @@
+const assert = require('assert');
+const crypto = require('crypto');
+const { series } = require('async');
+
+// Clients
+const { scalityS3Client, awsS3Client } = require('../../s3SDK');
+
+// Helpers
+const ReplicationUtility = require('../../ReplicationUtility');
+const { makeGETRequest, getResponseBody } = require('../../utils/request');
+
+const scalityUtils = new ReplicationUtility(scalityS3Client);
+const awsUtils = new ReplicationUtility(awsS3Client);
+
+// Vars
+const srcBucket = `source-bucket-${Date.now()}`;
+const destBucket = process.env.AWS_S3_BACKBEAT_BUCKET_NAME;
+const hex = crypto.createHash('md5')
+    .update(Math.random().toString())
+    .digest('hex');
+const keyPrefix = `${srcBucket}/${hex}`;
+const key = `${keyPrefix}/object-to-replicate-${Date.now()}`;
```

Comment:
Eventually I hope can move this key creation code into a helper util, but good for now! 😄